types/kue/index.d.ts

```diff
@@ -98,6 +98,7 @@ export declare class Job extends events.EventEmitter {
     ttl(param: any): Job;
     private _getBackoffImpl(): void;
     priority(level: string | number): Job;
+    priority(): number;
```

Comment:
The return value could either be a string. If you set the priority level to one of the map values, it is converted to number, but if you put your own custom string, you get it back as the same string.

Reply:
fixed